docs/src/conf.py

```diff
@@ -140,7 +140,7 @@
 # -- Options for HTMLHelp output ------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'Oriondoc'
+htmlhelp_basename = 'Oríondoc'
```

Comment:
Is that part of the URL? If so I would remove the accent.

Reply:
I agree